doc/reference/configuration/cfg_logging.rst

```diff
@@ -42,36 +46,41 @@
 
     Since version 1.7.4.
     By default, Tarantool sends the log to the standard error stream
-    (``stderr``). If ``log`` is specified, Tarantool sends the log to a file,
-    or to a pipe, or to the system logger.
+    (``stderr``). If ``log`` is specified, Tarantool can send the log to a ...
```

Comment:
Not sure that the list below improves the text significantly: all the options are just 1-2 words and don't have explanations.

Reply:
Anyway, in this case there are punctuation issues:
- `...` instead of `:` before the list
- `;` in the end of list items (see [our style guide](https://www.tarantool.io/en/doc/latest/contributing/docs/style/#ending-punctuation-in-lists-and-tables).

Reply:
> Not sure that the list below improves the text significantly: all the options are just 1-2 words and don't have explanations.

Generally, I agree but still think it's better than three `or` in a row :)